src/main/java/base/SheetsQuickstart.java

```diff
@@ -91,22 +91,21 @@ public void authenticate() throws IOException, GeneralSecurityException {
             System.out.println("No data found.");
         } else {
             for (List row : spreadsheetsValues) {
-               for(Object column : row) {
-                   if(row.indexOf(column) == 0) {
-                       selector = column.toString();
-                       System.out.println("SELECTOR " + selector);
-                   }
-                   if(row.indexOf(column) == 2) {
-                       copyOnSheets = column.toString();
-                       System.out.println("COPY ON SHEETS " + copyOnSheets);
-                   }
-               }
+                for(Object column : row) {
```

Comment:
Try to be consistent with code formatting I know it is annoying and we often forget to take care of it, I would say it is normal, but you can easily organise your code using the IDE functionalities: `cmd + alt + I` will indent lines automatically and `cmd + alt + L` will reformat your code, give it a try.